ui-frontend/packages/catalog-ui-search/src/main/webapp/component/visualization/maps/openlayers/drawing-and-display.tsx

```diff
@@ -170,12 +171,17 @@ const createGeoModel = (geo: GeometryJSON) => {
 }
 
 const modelToPolygon = (model: any): GeometryJSON | null => {
-  const coords = model.get('polygon')
+  let coords = model.get('polygon')
   if (
     coords === undefined ||
     validateGeo('polygon', JSON.stringify(coords))?.error
   ) {
-    return null
+    coords = [
```

Comment:
❓Does this cause something to be displayed on the map at 0,0 when you start drawing?

Reply:
You're right. The map would zoom in into existing shapes after opening, so I always missed that. Good catch!